plugins/jobs/src/js/JobsOverview.tsx

```diff
@@ -0,0 +1,116 @@
+import * as PropTypes from "prop-types";
+import * as React from "react";
+import { routerShape } from "react-router";
+// tslint:disable-next-line:no-submodule-imports
+import { BehaviorSubject } from "rxjs/BehaviorSubject";
+// tslint:disable-next-line:no-submodule-imports
+import "rxjs/add/observable/empty";
+
+import { graphqlObservable, componentFromStream } from "data-service";
+import gql from "graphql-tag";
+
+import JobsTabLoading from "./components/JobsTabLoading";
+import JobsTab from "./components/JobsTab";
+
+import JobModel, {
+  JobsQueryArgs,
+  JobConnection,
+  SortOption,
+  SortDirection
+} from "./data/JobModel";
+import JobTree from "#SRC/js/structs/JobTree";
+
+interface JobsOverviewProps {
+  params: { id: string };
+}
+
+const jobsOverviewQuery = (
+  filter: string,
+  sortBy: string,
+  sortDirection: string
+) => gql`
+  query {
+    jobs(filter: "${filter}", sortBy: "${sortBy}", sortDirection: "${sortDirection}") {
+      filteredCount
+      totalCount
+      nodes {
+        id
+        lastRunsSummary
+        lastRunStatus
+        scheduleStatus
+      }
+    }
+  }
+`;
+
+const DEFAULT_FILTERS: JobsQueryArgs = {
```

Comment:
todo: figure out how to reset the stream instead of setting its value back to some defaults